common/config/rush/pnpm-lock.yaml

```diff
@@ -7847,7 +7847,7 @@ packages:
   /rollup/1.27.14:
     dependencies:
       '@types/estree': 0.0.41
-      '@types/node': 13.1.2
+      '@types/node': 8.10.59
```

Comment:
I didn't see anything in the `pnpm` release notes, but I assume this was a bug fix in `pnpm`.  This dependency version was just updated to `13.1.2` in the last full update of `pnpm-lock.yaml` (with the previous version of `pnpm`).  Now it's going back to `8.10.59`.  The actual package specifies `"@types/node": "*"`, so I'm not sure which is more correct.

https://unpkg.com/browse/rollup@1.27.14/package.json

Reply:
8.10.59 seems correct given that rollup doesn't seem to care and we specify node 8 types elsewhere. I'm happy with this change.